mllib/src/main/scala/org/apache/spark/ml/source/libsvm/LibSVMRelation.scala

```diff
@@ -66,8 +76,63 @@ private[libsvm] class LibSVMRelation(val path: String, val numFeatures: Int, val
     case _ =>
       false
   }
+
+  override def prepareJobForWrite(job: _root_.org.apache.hadoop.mapreduce.Job):
+    _root_.org.apache.spark.sql.sources.OutputWriterFactory = {
```

Comment:
Can we use just `OutputWriterFactory` because of importing `org.apache.spark.sql.sources._`